pkg/apis/storage/types.go

```diff
@@ -672,3 +672,50 @@ type CSIStorageCapacityList struct {
 	// Items is the list of CSIStorageCapacity objects.
 	Items []CSIStorageCapacity
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// VolumeAttributesClass represents a specification of mutable volume attributes
+// defined by the storage driver. The class can be specified during dynamic provisioning
+// of PersistentVolumeClaims, and changed after provisioning.
+type VolumeAttributesClass struct {
+	metav1.TypeMeta
+
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta
+
+	// Name of the CSI driver
+	DriverName string
+
+	// parameters hold volume attributes defined by the CSI driver. These values
+	// are opaque to the system and are passed directly to the CSI driver.
```

Comment:
What does "the system" refer to here? Should we spell it out in more detail?

Reply:
It refers to the Kubernetes itself. I will change the word to Kubernetes from system later.